### ISSUE TITLE
Alt text is not rendered

### ISSUE THREAD

**Issue by edent** [Nexmo/nexmo-markdown-renderer#30]
None of the images on https://developer.nexmo.com have alt text. This is a WCAG violation.

See https://github.com/Nexmo/nexmo-developer/issues/3238

Responsible code appears to be at https://github.com/Nexmo/nexmo-markdown-renderer/blob/master/lib/nexmo_markdown_renderer/filters/markdown_filter.rb#L62

It fails to parse `![text like this](image.png)`

It should also emit a blank alt text if none is present. At the moment it literally prints `#{alt_text}`
![Screenshot from 2020-10-05 22-42-27](https://user-images.githubusercontent.com/837136/95136242-acf4a800-075d-11eb-88c1-e1bf7bb2a930.png)

I don't know enough Ruby to correct this.

**Comment by hummusonrails** [comment]
Thank you, @edent, for raising this. We really appreciate calling attention to it!
